documentation/src/main/asciidoc/reference/backend-elasticsearch.asciidoc

```diff
@@ -712,6 +712,23 @@ include::{resourcesdir}/analysis/elasticsearch-simple.properties[]
 <1> Assign the configurer to the backend using a Hibernate Search configuration property.
 ====
 
+[NOTE]
+====
+A different analysis configurer can be assign to each index:
+
+[source]
+----
+# To set the default configurer for all indexes:
+hibernate.search.backend.analysis.configurer com.mycompany.MyAnalysisConfigurer
+# To assign a specific configurer to a specific index:
+hibernate.search.backend.indexes.<index name>.analysis.configurer com.mycompany.MySpecificAnalysisConfigurer
+----
+
+If a specific configurer is assigned to an index,
+the default configurer will be ignored for that index:
+only definitions from the specific configurer will be taken into account.
```

Comment:
If the user needs to reuse definitions in the backend configurer to a specific index configurer, class inheritance can be used. Or cannot it?

Reply:
Sure. Inheritance, composition, whatever. It's just code.